foundation-extensions/lifecycle-for-edge-network/README.md

```diff
@@ -78,16 +78,13 @@ After you complete your configuration, verify that your rule looks like the foll
 
 ![](../../.gitbook/assets/rulecomplete-forward-lifecycle-to-edge-network.png)
 
-1. Click **Save**
-2. Rebuild your Experience Platform Launch property and deploy it to the correct Environment.
+1. Click **Save**.
+2. Rebuild your Experience Platform tags property and deploy it to the correct environment.
 
-## Add the AEP Lifecycle extension to your app
+## Add the Lifecycle extension to your app
 
 For implementation details, please reference:
-1. [Add Lifecycle to your app](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/lifecycle#add-lifecycle-to-your-app)
-2. [Register Lifecycle with Mobile Core and add appropriate Start/Pause calls](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/lifecycle#register-lifecycle-with-mobile-core-and-add-appropriate-start-pause-calls)
-
-In addition, use the following steps to [add the AEP Edge extension to your app](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension#add-the-aep-edge-extension-to-your-app).
-
-
+1. [Add Lifecycle to your app](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/lifecycle#add-lifecycle-to-your-app).
+2. [Register Lifecycle with Mobile Core and add appropriate Start/Pause calls](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/lifecycle#register-lifecycle-with-mobile-core-and-add-appropriate-start-pause-calls).
 
+In addition, use the following steps to [add the Edge extension to your app](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension#add-the-aep-edge-extension-to-your-app).
```

Comment:
- add the Edge Network extension to your app
- should rename this anchor as well add-the-edge-network-extension-to-your-app)